Formplayer/testPages/webapps/webapps_basics.py

```diff
@@ -0,0 +1,214 @@
+import time
+from random import randint
+from datetime import datetime
+
+from selenium.webdriver.common.by import By
+from HQSmokeTests.userInputs.generate_random_string import fetch_random_string
```

Comment:
Replace HQSmokeTests to Formplayer

Reply:
Replaced all HQSmokeTests to Formplayer in all files.